OpenLIFULib/OpenLIFULib/lazyimport.py

```diff
@@ -5,7 +5,7 @@
 import slicer
 import importlib
 import sys
-from OpenLIFULib.busycursor import BusyCursor
+from OpenLIFULib.util import BusyCursor
```

Comment:
Huh, I wonder why this bug wasn't causing problems before

Reply:
Yeah its weird. It was causing issues with the custom app build.